core/modes/sleep_mode_not_deep_sleep.talon

```diff
@@ -0,0 +1,8 @@
+mode: sleep
```

Comment:
sleep_mode_dragon

Reply:
Given that the file is for wake up commands available with and without dragon, I renamed this to sleep_mode_wakeup.talon for now. Is that acceptable? I renamed the file dealing with dragon sleep commands "sleep_mode_dragon.talon".